src/pages/Survey/SurveyBox/Loading.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+
+import styled, { keyframes } from 'styled-components';
+
+function Loding() {
```

Comment:
```suggestion
function Loading() {
```
오타도 휴먼 에러를 유발할 수 있는 부분이기 때문에 발견 할 때마다 고쳐주세요